dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -24,5 +24,70 @@
+# All messages must use the range CWWKD1000 to CWWKD1399
+
+CWWKD1000.repo.general.err=CWWKD1000E: The {0} method of the {1} repository \
+ interface encountered an error indicating that the repository method might \
+ not be valid. The error is: {2}.
+CWWKD1000.repo.general.err.explanation=An error was found with the specified \
+ repository method.
+CWWKD1000.repo.general.err.useraction=Locate the repository interface and method \
+ and determine whether corrections are needed.
+
+CWWKD1001.no.primary.entity=CWWKD1001E: An entity class is needed by the {0} \
+ repository method, but the {1} repository interface does not specify a primary \
+ entity class. To correct this, have the {1} repository interface extend one of \
+ the built-in repository supertypes, such as {2}, supplying the entity class as \
+ the first type parameter.
+CWWKD1001.no.primary.entity.explanation=Repository methods without an entity \
+ class of their own require a primary entity class to be defined by the \
+ repository.
+CWWKD1001.no.primary.entity.useraction=Define a primary entity class for the \
+ repository.
+
+CWWKD1002.method.annos.err=CWWKD1002E: The {0} method of the {1} repository \
+ interface is annotated with a combination of annotations, {2}, that is not \
+ valid.
+CWWKD1002.method.annos.err.explanation=The combination of annotations is not \
+ valid on a single repository method.
+CWWKD1002.method.annos.err.useraction=Locate the repository interface and method \
+ and determine the correct method annotations to use.
+
+CWWKD1003.lifecycle.rtrn.err=CWWKD1003E: The {0} return type of the {1} method \
+ of the {2} repository interface is not a valid return type for a repository \
+ {3} method. Valid return types include: {4}.
+CWWKD1003.lifecycle.rtrn.err.explanation=The return type of the method is not \
+ one of the supported return types for Insert, Update, and Save methods.
+CWWKD1003.lifecycle.rtrn.err.useraction=Refer to the Jakarta Data JavaDoc of the \
+ Insert, Update, and Save methods for valid return types.
+
+CWWKD1004.general.rtrn.err=CWWKD1004E: The {0} return type of the {1} method \
+ of the {2} repository interface is not a valid return type for a repository \
+ method.
+CWWKD1004.general.rtrn.err.explanation=The return type of the method is not \
+ one of the supported return types for the type of repository method.
+CWWKD1004.general.rtrn.err.useraction=Refer to the Jakarta Data JavaDoc for a \
+ list of valid return types for the type of repository method.
+
+CWWKD1005.find.rtrn.err=CWWKD1005E: The {0} method of the {1} repository \
+ interface specifies the {2} return type, which is not convertible from the \
+ {3} entity type. The result type of a find operation must be the \
+ entity type, the type of an entity property, a Java record composed of \
+ entity properties, or be constructed by JPQL that is supplied to the \
+ Query annotation. The return type of the repository find method can be an \
```

Comment:
> The return type of the repository find method can be an \
 Optional or single value of the result type or it can be one of {4} or an \
 array of the result type, _allowing multiple results to be returned_.
 
 What allows multiple results here? Is it specifically the array or is it the entire combination of possible values listed in this sentence?